### ISSUE TITLE
关于剪枝代码的疑虑

### ISSUE THREAD

**Issue by Ixuanzhang** [Eric-mingjie/network-slimming#52]
![Screenshot from 2020-07-15 01-00-52](https://user-images.githubusercontent.com/8215764/87454541-afe86c80-c636-11ea-9341-c74435d7d92e.png)
你好，针对原来的模型，按照上述代码，仅涉及到了nn.BatchNorm2d、nn.Conv2d、nn.Linear三种情况，并没考虑到m0为Bottleneck的情况，故这样子写，并没有剪枝Bottleneck中的BN层。不知道是我理解错了，还是？

**Comment by Eric-mingjie** [comment]
请问是哪个文件里的?

**Comment by Ixuanzhang** [comment]
resprune.py中  第144行开始

**Comment by Eric-mingjie** [comment]
https://github.com/Eric-mingjie/network-slimming/blob/master/resprune.py#L153

**Comment by Ixuanzhang** [comment]
我知道那句注释   但因为缺少 elif isinstance(m0, Bottleneck)  我觉得对Bottleneck中的BN层并未作处理     
因为当 
model = resnet101()
for id,m in enumerate(model.modules()):
    print(id)
    print(m)

结果是：
![re](https://user-images.githubusercontent.com/8215764/87505565-bd840d80-c69b-11ea-9ecc-71553d8b080a.png)

所以按照我的理解，您写的代码中未考虑Bottleneck中的BN。
可能是我理解错了，望指教！谢谢

**Comment by Ixuanzhang** [comment]
我发现我理解的有问题   我再详细看一下  谢谢
